router/pkg/config/config.schema.json

```diff
@@ -624,6 +624,30 @@
       "default": "/",
       "description": "The path of the GraphQL Playground. The GraphQL Playground is a web-based GraphQL IDE that allows you to interact with the GraphQL API. The default value is '/'."
     },
+    "file_uploads": {
```

Comment:
Use singular it is the feature "file_upload".

Reply:
Done